packages/x-date-pickers/src/YearCalendar/pickersYearClasses.ts

```diff
@@ -24,10 +20,8 @@ export function getPickersYearUtilityClass(slot: string) {
   return generateUtilityClass('MuiPickersYear', slot);
 }
 
-export const pickersYearClasses = generateUtilityClasses('MuiPickersYear', [
+export const pickersYearClasses = generateUtilityClasses<PickersYearClassKey>('MuiPickersYear', [
   'root',
-  'modeDesktop',
-  'modeMobile',
```

Comment:
This and same change in `pickersMonthClasses` essentially makes it a public API breaking change.
We should probably add some breaking change doc and mention in PR..? 🤔

Reply:
Yes, I was waiting for us to agree on the change before discussing the migration guide.

I think this class make no real sense (why put it on those elements and not on others ?)

Reply:
> I think this class make no real sense (why put it on those elements and not on others ?)

I really don't know as well... Maybe there once was a request to allow for the items per row to be easily controlled via CSS? 🤔